src/django_safemigrate/management/commands/safemigrate.py

```diff
@@ -101,13 +97,13 @@ def pre_migrate_receiver(self, *, plan, **_):
             return  # Only run once
         self.receiver_has_run = True
 
-        safemigrate_state = safemigrate()
-        if safemigrate_state == SafeMigrate.disabled:
+        safemigrate_state = safemigrate_mode()
```

Comment:
I feel like `safemigrate_state` should be renamed to match mode rather than state.